src/Components/Facility/ConsultationForm.tsx

```diff
@@ -187,11 +235,17 @@ export const ConsultationForm = (props: any) => {
       setIsLoading(true);
       const data = {
         symptoms: state.form.symptoms,
-        other_symptoms: state.form.otherSymptom ? state.form.other_symptoms : undefined,
-        symptoms_onset_date: state.form.hasSymptom ? state.form.symptoms_onset_date : undefined,
+        other_symptoms: state.form.otherSymptom
+          ? state.form.other_symptoms
+          : undefined,
+        symptoms_onset_date: state.form.hasSymptom
+          ? state.form.symptoms_onset_date
+          : undefined,
         suggestion: state.form.suggestion,
         admitted: JSON.parse(state.form.admitted),
-        admitted_to: JSON.parse(state.form.admitted) ? state.form.admitted_to : undefined,
+        admitted_to: JSON.parse(state.form.admitted)
```

Comment:
Codacy found an issue: [Identifier 'admitted_to' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808151)